html/settings/view/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>ScoreBoard View Settings</title>
```

Comment:
Just ScoreBoard Settings here and in the index would make more sense, given the left column affects non-view things.

Reply:
That's done now. (I've shortened to 'Settings' for the operator interface.)